conf/defaults.ini

```diff
@@ -163,12 +163,17 @@ expect_continue_timeout_seconds = 1
 # A value of zero (0) means no limit.
 max_conns_per_host = 0
 
+# Deprecated, use dataproxy_max_idle_connections
 # The maximum number of idle connections that Grafana will keep alive.
 max_idle_connections = 100
 
+# Deprecated, use dataproxy_max_idle_connections
```

Comment:
This one we can remove completely from the config file. If anyone has a custom config using this we'll log a depreciation warning. Similar to https://github.com/grafana/grafana/blob/31aaf94e91a15bb5e87fa89cf74ea3d542f383c6/pkg/setting/setting.go#L1238